### ISSUE TITLE
Cannot highlight links in response section. Clicking on them opens the link in a Tab

### ISSUE THREAD

**Issue by markh1** [a85/POSTMan-Chrome-Extension#6984]
I have postman Version 7.3.2
win32 10.0.17763 / x64.

Ever since the last update, I cannot highlight (and copy to windows clipboard) a url (contained in an xml comment tag) within the xml response returned from my api request.

For example:
```
<?xml version="1.0" encoding="UTF-8"?>
<!--https://myapisite.com/myrequest/1011/-->
... remaining xml ...
```

All of my requests include the url that was received in the returned response (inside a comment).  This allows me to verify the url that was sent (after the variables in the postman request were resolved).

However, when i try to highlight the url, as soon as i let up on the mouse, postman treats it as a "click", and opens the url in a separate window.  This behavior started with the last update.  I used to be able to highlight and copy a url within a comment tag, without it being treated as a click.

**Comment by motenzhang** [comment]
![image](https://user-images.githubusercontent.com/2246795/62112581-8d562a00-b2e5-11e9-96d4-a5fc3921cb4c.png)
yes, revert to the previous one or waiting to could switch response editor

**Comment by kamalaknn** [comment]
This seems to have started with our recent changes to the response viewer. We have marked this as a bug. We will keep this thread updated on the status of the fix.

**Comment by kamalaknn** [comment]
This is fixed on the latest version of Postman.

**Comment by ericzhao007** [comment]
In version 9.3.1, the problem is not fixed

**Comment by thachvan** [comment]
Fixed on 10.0.20
